nemo/collections/asr/parts/submodules/multi_head_attention.py

```diff
@@ -129,9 +129,7 @@ def forward(self, query, key, value, mask, pos_emb=None, cache=None, cache_next=
         returns:
             output (torch.Tensor): transformed `value` (batch, time1, d_model) weighted by the query dot key attention
         """
-        key, value, query, cache_next = self.update_cache(
-            key=key, value=value, query=query, cache=cache, cache_next=cache_next
-        )
+        key, value, query = self.update_cache(key=key, value=value, query=query, cache=cache, cache_next=cache_next)
```

Comment:
How comes no test failed due to this? do we have a test that runs this part of the code?